### PR TITLE
Warn admins about active Internet sounds

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

For various reasons an admin may not be aware there is currently an Internet sound playing. (Usually TGUI related or just recently logged in.)

This provides a warning and optional override if they try to start an Internet sound while one is already running.

https://github.com/tgstation/tgstation/assets/83487515/af4197c9-bf9d-43e4-a407-a556b88a6d30

## Why It's Good For The Game

Less "HEY I WAS ALREADY PLAYING SOMETHING!"

## Changelog

:cl: LT3
admin: Admins are now warned if they are going to override a currently playing admin midi
/:cl:

### CODE REVIEW

**Comment by dragomagol** [comment]
Yessssss

**Comment by ShizCalev** [comment]
bless

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -212,6 +221,10 @@
 		SEND_SOUND(M, sound(null))
 		var/client/C = M.client
 		C?.tgui_panel?.stop_music()
+
+	if(S_TIMER_COOLDOWN_TIMELEFT(SStimer, COOLDOWN_INTERNET_SOUND))
+		S_TIMER_COOLDOWN_RESET(SStimer, COOLDOWN_INTERNET_SOUND)
```
Comment (san7890) [inline code/modules/admin/verbs/playsound.dm:226]: should this really be a conditional reset or should we just always do this?

Comment (lessthnthree) [reply code/modules/admin/verbs/playsound.dm:226]: We may not, I wasn't sure if we'd throw a runtime trying to reset a timer that isn't running. I'll test it.

Comment (lessthnthree) [reply code/modules/admin/verbs/playsound.dm:226]: Yeah it's fine to always run.

**Pull request merged**